bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/api/rest/ListableRest.java

```diff
@@ -62,7 +62,7 @@ public List<T> getDevices() throws FreeboxException {
             if (result != null) {
                 return result;
             }
-            throw new FreeboxException("Devicelist is empty");
```

Comment:
List is not present in the Freebox API success response (for repeater). So it is not an error.